pro/src/pages/Reimbursements/BankInformations/BankInformations.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react'
+
+import { api } from 'apiClient/api'
+import { GetOffererBankAccountsResponseModel } from 'apiClient/v1'
+import { useReimbursementContext } from 'context/ReimbursementContext/ReimbursementContext'
+import fullLinkIcon from 'icons/full-link.svg'
+import fullMoreIcon from 'icons/full-more.svg'
+import { Button, ButtonLink } from 'ui-kit'
+import { ButtonVariant } from 'ui-kit/Button/types'
+import Spinner from 'ui-kit/Spinner/Spinner'
+
+import styles from './BankInformations.module.scss'
+
+const BankInformations = (): JSX.Element => {
+  const { selectedOfferer } = useReimbursementContext()
+
+  const [isOffererBankAccountsLoading, setIsOffererBankAccountsLoading] =
+    useState<boolean>(false)
+  const [, setSelectedOffererBankAccounts] =
+    useState<GetOffererBankAccountsResponseModel | null>(null)
+
+  useEffect(() => {
+    const getSelectedOffererBankAccounts = async (
+      selectedOffererId: number
+    ) => {
+      setIsOffererBankAccountsLoading(true)
+      try {
+        const offererBankAccounts =
+          await api.getOffererBankAccountsAndAttachedVenues(selectedOffererId)
+        setSelectedOffererBankAccounts(offererBankAccounts)
+        setIsOffererBankAccountsLoading(false)
+      } catch (error) {
+        setIsOffererBankAccountsLoading(false)
+      }
+    }
+
+    if (selectedOfferer !== null) {
+      getSelectedOffererBankAccounts(selectedOfferer.id)
+    }
```

Comment:
Pour rester cohérent avec l'existant
```suggestion
    selectedOfferer && getSelectedOffererBankAccounts(selectedOfferer.id)
```